client/src/http_sender.rs

```diff
@@ -27,6 +27,13 @@ impl HttpSender {
     }
 }
 
+#[derive(Deserialize, Debug)]
+struct RpcErrorObject {
+    code: i64,
+    message: String,
+    /*data field omitted*/
```

Comment:
One day it'd be cool to return a full `RpcCustomError` object here.  But for now I'm content with just having access to the error `code`